_posts/2016-05-12-Developers-love-spaces.md

```diff
@@ -0,0 +1,54 @@
+---
+layout: post
+title: Developers love spaces - Solita Developer Survey 2016 results
+author: arto
+excerpt: Now we know what developers love and love not this year
+categories: 
+- Java
+tags: 
+- Java
+- Clojure 
+- Javascript
+- Development
+- Culture
+- Solita
+---
+
+## What do devs love?
+
+Well a lot of things, apparently. Today, being a dev is about being passionate about things. But in this case, I think I already let out a spoiler in the title. Devs do love spaces. And, thanks to recent results from Solita Developer Survey, I got the stats to prove it. According the the survey, crushing percentage of 72,2% of all solita devs indent their code with spaces, as opposed to tabs. 2% didn't understand the question, so I suppose they don't indent.
```

Comment:
According the the -> to the
*a* crushing percentage
72,2 -> 72.2% (English decimal sep)
solita -> Solita